### ISSUE TITLE
lfs with minio fails with big files

### ISSUE THREAD

**Issue by Zokormazo** [go-gitea/gitea#28494]
### Description

I'm using gitea 1.21.2 with lfs backed in wasabi using minio storage. Pushing small files up to 100mb is working as expected, but bigger ones are failng and `git push` stuck uploading forever. Logs are showing a content length and body length mismatch, not sure if that's expected

### Gitea Version

1.21.2

### Can you reproduce the bug on the Gitea demo site?

No

### Log Gist

https://gist.github.com/Zokormazo/a6be870ce28ae80d6fefd79aff06be9d

### Operating System

rhel 8

### How are you running Gitea?

rootless container

### Database

PostgreSQL

**Comment by KN4CK3R** [comment]
Some kind of proxy between Gitea and the storage which cancels the connection?

**Comment by Zokormazo** [comment]
Nope, I can upload 1gb+ files using `public.ecr.aws/aws-cli/aws-cli` from the same host without issues
